content/chainguard/chainguard-images/features/cve_visualizations/index.md

```diff
@@ -13,59 +13,88 @@ tags: ["CONCEPTUAL", "CHAINGUARD IMAGES", "PRODUCT"]
+
+You can find this same comparison data when navigating to a specific image in either the **Browse Images** section or in your **Organization Images**. After navigating to either of these sections, click on or search for any image you like.
+
+By default, you will be taken to the image's **Versions** tab. Click on the **Comparison** tab at the far right. There, you'll be presented with the same comparison information found in the **Reports** section. At the top are some control menus, allowing you to select the date range for the comparison and, if available, the alternative you'd like to compare the Chainguard Image against. This example shows the PHP image:
+
+<center><img src="cve-viz-5.png" alt="Screenshot of the Chainguard PHP image's Comparison tab in the Browse Images section of the Chainguard Console, with data showing how it compares against the php:latest image." style="width:1100px;"></center>
+<br /> 
+
+
+## Accessing CVE Visualizations in the Images Directory
+
+Similar to the CVE reports found in the **Browse Images** and **Organization Images** section of the Chainguard Console, you can find CVE reports for every one of Chainguard's container images in the [Images Directory](https://images.chainguard.dev/).
+
+After navigating to the directory, click on or search for any image you like. Again, you will be taken to the image's **Versions** tab by default. Click on the **Comparison** tab at the right to view the CVE Comparison data. This example shows the nginx image:
+
+<center><img src="cve-viz-6.png" alt="Screenshot from Chainguard's Public Images Directory showing a portion of the Chainguard nginx image's Comparison tab, with data showing how it compares against the nginx:alpine image." style="width:1100px;"></center>
+<br /> 
 
-You will also see the same comparison data when navigating to a specific image in your Organization
-images, under the "Comparison" tab.
 
 ## Limitations
 
-We only have data for a limited subset of alternative images. We will expand this set over time, but
-please let us know of any particular comparison data you are looking for.
+Some images do not currently have a comparative alternative. In these cases, the Comparison report will only show data for the Chainguard image. 
 
 ## Learn More
 
-The CVE data used in these reports is from the [Grype vulnerability
-scanner](https://github.com/anchore/grype). Vulnerability data is constantly evolving, so we scan
-images each day and store the results. The results shown are the vulnerabilities found on the day in
-question; scanning the images again with a newer database will show different results.
+The CVE data used in these reports is from the [Grype vulnerability scanner](https://github.com/anchore/grype). Vulnerability data is constantly evolving, so we scan images each day and store the results. The results shown are the vulnerabilities found on the day in question; scanning the images again with a newer database will show different results.
```

Comment:
I would include a link to our [Grype tutorial](https://edu.chainguard.dev/chainguard/chainguard-images/staying-secure/working-with-scanners/grype-tutorial/).